### PR TITLE
Ignore folders

### PR DESCRIPTION
**Pull request by laureysruben**
Check if the current file is a folder, and if so do not try to add headers. Trying to add the headers on folders gives errors...

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -31,25 +32,27 @@ module.exports = function (headerText, data) {
...
     }
 
-    var template = data === false ? headerText : gutil.template(headerText, extend({file : file, filename: filename}, data));
+    var template = data === false ? headerText : gutil.template(headerText, extend({ file: file, filename: filename }, data));
     concat = new Concat(true, filename);
 
-    if (file.isBuffer()) {
-      concat.add(null, new Buffer(template));
-    }
+    if (!fs.lstatSync(file.path).isDirectory()) {
```
Comment (tracker1) [inline index.js]: return early instead of wrapping everything...

**Comment by tracker1** [comment]
return early instead of wrapping... also, fix the failing test, and I'll approve the PR and push out a point release.

**Comment by laureysruben** [comment]
I changed the code to return early, but the test fails because of a missing file. (file2.txt) Is it supposed to error when a file does not exist, or should it be caught in some way?

**Comment by tracker1** [comment]
I went ahead and merged, will see about the test errors if I can remember to do that tonight

**Comment by ibc** [comment]
Please, revert this commit. It breaks usage with in-memory streams: #37

File: index.js [hunk 2]
```diff
@@ -31,9 +32,18 @@ module.exports = function (headerText, data) {
       filename = '';
     }
 
-    var template = data === false ? headerText : gutil.template(headerText, extend({file : file, filename: filename}, data));
+    var template = data === false ? headerText : gutil.template(headerText, extend({ file: file, filename: filename }, data));
     concat = new Concat(true, filename);
 
+    if (fs.lstatSync(file.path).isDirectory()) {
```
Comment (sym3tri) [inline index.js]: This broke our build :cry: 
b/c it's stating a file that does not exist and there is no try/catch around this check.

**Comment by stevenvachon** [comment]
What if the file has not yet been written? I'm getting errors as a result of this change.

**Pull request merged**